tests/test_citations.py

```diff
@@ -12,7 +14,7 @@
 
 class CitationsUtilsTestCase(OsfTestCase):
     def test_datetime_to_csl(self):
-        # Convert a datetime instance to csl's date-variable schema
+        """Convert a datetime instance to csl's date-variable schema"""
```

Comment:
Prefer code comments to docstrings in tests.

> Use long, descriptive names. This often obviates the need for doctrings in test methods. This also makes it easier to locate tests that fail.

http://cosdev.readthedocs.org/en/latest/process/testing.html#testing